langwatch/src/pages/onboarding/organization.tsx

```diff
@@ -55,28 +39,64 @@ import { LoadingScreen } from "../../components/LoadingScreen";
 import { Checkbox } from "../../components/ui/checkbox";
 import { Link } from "../../components/ui/link";
 import { RadioGroup } from "../../components/ui/radio";
-import { toaster } from "../../components/ui/toaster";
 import { usePublicEnv } from "../../hooks/usePublicEnv";
 import { useRequiredSession } from "../../hooks/useRequiredSession";
 import { titleCase } from "../../utils/stringCasing";
 import { useOrganizationTeamProject } from "../../hooks/useOrganizationTeamProject";
+import { LuBinoculars, LuCode, LuCpu, LuEarth, LuExternalLink, LuEye, LuFlaskConical, LuFlaskRound, LuMicroscope, LuPresentation, LuProjector, LuShare2, LuUserSearch, LuWandSparkles } from "react-icons/lu";
+import { toaster } from "~/components/ui/toaster";
 
 type OrganizationFormData = {
   organizationName: string;
   phoneNumber: string;
   terms: boolean;
   usage: string;
   solution: string;
-  projectType: string;
   companySize: string;
-  howDidYouHearAboutUs: string;
-  companyType: string;
-  otherCompanyType: string;
-  otherProjectType: string;
-  otherHowDidYouHearAboutUs: string;
+  featureUsage: string;
+  yourRole: string;
   utmCampaign: string | null;
 };
 
+const options = {
+  "For my company": <Briefcase size={16} color="orange" />,
+  "For my clients": <Users size={16} color="orange" />,
+  "For myself": <User size={16} color="orange" />,
+};
+
+const langwatchSolution = {
+  "SaaS": <Cloud size={16} color="orange" />,
+  "On-Premise": <Server size={16} color="orange" />,
+};
+
+const featureUsage: Record<string, React.ReactNode> = {
+  "Evaluating": <LuMicroscope size={16} color="orange" />,
+  "Everything": <LuEarth size={16} color="orange" />,
+  "Model/Prompt experimentation": <LuFlaskConical size={16} color="orange" />,
+  "Monitoring & Observability": <LuBinoculars size={16} color="orange" />,
+  "Collaboration & annotation": <LuShare2 size={16} color="orange" />,
+  "Just looking": <LuEye size={16} color="orange" />,
+};
+
+const roles: Record<string, React.ReactNode> = {
+  "AI Engineer": <LuWandSparkles size={16} color="orange" />,
+  "Data Scientist": <LuFlaskRound size={16} color="orange" />,
+  "Software Engineer": <LuCode size={16} color="orange" />,
+  "Product Manager": <LuPresentation size={16} color="orange" />,
+  "Engineering Manager": <LuProjector size={16} color="orange" />,
+  "CTO/Founder": <LuCpu size={16} color="orange" />,
+  "Other": <LuUserSearch size={16} color="orange" />,
+};
```

Comment:
nit: instead of repeating size={16} and color="orange" everywhere you could add a prop `_icon={{width: "16px", height: "16px", color: "orange.400"}` or something on the HStack they are used

Reply:
I was just matching the pattern that was already here to be consistent. Can change for all but would rather get this out asap

Reply:
Your call tho, will change if you prefer